src/pwnedPassword.js

```diff
@@ -1,48 +1,20 @@
-import fetchFromApi from './internal/pwnedpasswords/fetchFromApi';
+import sha1 from 'js-sha1';
+import pwnedPasswordRange from './pwnedPasswordRange';
 
 /**
  * Fetches the pwned status for the given password, indicating whether or not it
- * has been previously exposed in a breach. Passwords can be plain text or a
- * SHA1 hash. The remote API will automatically attempt to discern between the
- * two, but in the case where the password you wish to check is actually a
- * plain text string containing a hash and you don't want the API to treat it as
- * a hash, you can override the auto detection behavior by setting the isAHash
- * option to true.
+ * has been previously exposed in a breach. The password is given in plaintext,
+ * but only the first 5 characters of its SHA1 hash will be sumitted to the API.
+ * The final evalution will be done locally.
  *
- * @param {string} password a password (plain text string or SHA1 hash)
- * @param {Object} [options] a configuration object
- * @param {boolean} [options.isAHash] the pre-hashed password is a hash
- * (default: false)
- * @returns {Promise} a Promise which resolves to true if the given password has
- * been exposed in a breach (or false if not), or rejects with an Error
+ * @param {string} password a password in plaintext
+ * @returns {Promise} a Promise which resolves to the number of times the password has
+ * been breached, or rejects with an Error
  * @example
```

Comment:
Can we change the `@returns` verbiage here to `a Promise which resolves to the number of times the password has been exposed in a breach, or rejects with an Error`?